packages/thumbprint-react/components/Popover/index.jsx

```diff
@@ -0,0 +1,219 @@
+import React, { useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Manager, Reference, Popper } from 'react-popper';
+import onClickOutside from 'react-onclickoutside';
+import startsWith from 'lodash/startsWith';
+
+import * as tokens from '@thumbtack/thumbprint-tokens';
+
+import canUseDOM from '../../utils/can-use-dom';
+
+import { TextButton } from '../Button/index.jsx';
+import { Themed } from '../UIAction/index.jsx';
+import { NavigationCloseTiny } from '../../icons/index.jsx';
+
+import styles from './index.module.scss';
+
+const ESC_KEY = 27;
+
+// Internal component only. Proptypes are defined for the main component `Popover` at the end of the
+// file.
+// eslint-disable-next-line react/prop-types
```

Comment:
Let's also add `PropTypes` for the internal one. (I don't mind if we disable for tests, but I do think it's good to have for internal components.)

Reply:
i agree you shouldn't have undocumented props, but in this case they're identical to the ones below, which is why i left a comment. anyone who wants to know what to pass in here can easily read those proptypes. IMO duplicating all of them is worse, it just adds visual noise to the file